prefq/examples/imitation_preference_comparisons.py

```diff
@@ -34,6 +31,8 @@
 from stable_baselines3 import PPO
 from stable_baselines3.common.evaluation import evaluate_policy
 
+from prefq.examples.queryclient import QueryClient
```

Comment:
I think the `queryclient` module should not be part of `examples`. I'd also prefer to use an underscore in the name: I think  `prefq.query_client` would be good.